make/addons.mk

```
@@ -84,3 +84,11 @@ generate-mindthegap-repofile: generate-helm-configmap ; $(info $(M) generating h
 .PHONY: template-helm-repository
 template-helm-repository: generate-mindthegap-repofile ## this is used by gorealeaser to set the helm value to this.
 	sed -i '/RepositoryURL:/s#\(RepositoryURL: *\)\(.*\)#\1{{ if .Values.selfHostedRegistry }}oci://helm-repository.{{ .Release.Namespace }}.svc/charts{{ else }}\2{{ end }}#'  "./charts/cluster-api-runtime-extensions-nutanix/templates/helm-config.yaml"
+
+.PHONY: list-images
+list-images:
+	cd hack/tools/fetch-images && go build
+	./hack/tools/fetch-images/fetch-images \
+		-chart-directory=./charts/cluster-api-runtime-extensions-nutanix/ \
+		-helm-chart-configmap=./charts/cluster-api-runtime-extensions-nutanix/templates/helm-config.yaml \
+		-caren-version=$(CAREN_VERSION) >> caren-images.txt
```

What do you think about letting the caller decide what to do with the output? 
```suggestion
		-caren-version=$(CAREN_VERSION)
```

---

its only needed by the release process so this isn't necessary.